Bottom: cf1a76be41e7278a3171df59730d867fd67a48a7
Top:    68992b7ff032e5f99c73e4da0aaefcc79ee819ec
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-06 00:39:06 +0100

Refresh of hid-gtk-connect-up-the-file-ch

---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index 72cd411..aa33d85 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -2169,8 +2169,8 @@ hid_gtk_init ()
   ghid_hid.flush_debug_draw         = ghid_flush_debug_draw;
   ghid_hid.finish_debug_draw        = ghid_finish_debug_draw;
 
-  ghid_hid.notify_save_pcb            = ghid_notify_save_pcb;
-  ghid_hid.notify_pcb_filename_change = ghid_hid.notify_pcb_filename_change;
+  ghid_hid.notify_save_pcb          = ghid_notify_save_pcb;
+  ghid_hid.notify_filename_changed  = ghid_hid_notify_filename_changed;
 
   hid_register_hid (&ghid_hid);
 #include "gtk_lists.h"
diff --git a/src/hid/gtk/gui-top-window.c b/src/hid/gtk/gui-top-window.c
index d72be88..4ba598c 100644
--- a/src/hid/gtk/gui-top-window.c
+++ b/src/hid/gtk/gui-top-window.c
@@ -420,8 +420,7 @@ ghid_notify_save_pcb (const char *filename, bool done)
 }
 
 void
-ghid_notify_pcb_filename_changed (const char *old_filename,
-                                  const char *new_filename)
+ghid_notify_filename_changed (void)
 {
   disconnect_file_change_monitor (ghidgui);
   connect_file_change_monitor (ghidgui);
